CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ should change the heading of the (upcoming) version to include a major version b
 - Upgraded from the `1.x` to `2.x` version of `semantic-ui-react`
 - Added support for `schema.examples` in the material ui theme fixing (https://github.com/rjsf-team/react-jsonschema-form/issues/2368, https://github.com/rjsf-team/react-jsonschema-form/issues/2557)
 
+## @rjsf/bootstrap-4
+- Avoid importing the whole of `react-icons` (https://github.com/rjsf-team/react-jsonschema-form/pull/3046)
```

Comment:
If you prefer:

```suggestion
- Avoid importing the whole of `react-icons` (https://github.com/rjsf-team/react-jsonschema-form/pull/3046, https://github.com/react-icons/react-icons/issues/154)
```

Reply:
@fregante Yes, that suggestion works, go ahead and commit it along with the updates to the snapshots with the new icons.